@Sangarshanan it seems that one test is failing:
```
_ TestStatNameHandlerNotSupportedRule.test_check_failed[plugins0--stat_name_handler field is no longer supported in AirflowTestPlugin. stat_name_handler option in [scheduler] section in airflow.cfg should be used to achieve the same effect.] _

self = <tests.upgrade.rules.test_stat_name_handler_not_supported.TestStatNameHandlerNotSupportedRule object at 0x7f6224a8ca50>
plugins = [<class 'tests.upgrade.rules.test_stat_name_handler_not_supported.MockPluginWithStatNameHandler'>, <class 'tests.plugins.test_plugin.MockPluginA'>]
test_config = ''
expected = 'stat_name_handler field is no longer supported in AirflowTestPlugin. stat_name_handler option in [scheduler] section in airflow.cfg should be used to achieve the same effect.'

    @pytest.mark.parametrize(
        'plugins, test_config, expected',
        [
            (PLUGINS_WITH_STAT_NAME_HANDLER, "", CHECK_ERR_MSG),
            (PLUGINS_WITHOUT_STAT_NAME_HANDLER, "", None),
            (PLUGINS_WITHOUT_STAT_NAME_HANDLER, TEST_CONFIG, None),
            (PLUGINS_WITH_STAT_NAME_HANDLER, TEST_CONFIG, None),
        ]
    )
    def test_check_failed(self, plugins, test_config, expected):
        rule = StatNameHandlerNotSupportedRule()

        assert isinstance(rule.description, str)
        assert isinstance(rule.title, str)

        test_conf = AirflowConfigParser()
        test_conf.read_string(test_config)

        mock_conf = mock.patch("airflow.upgrade.rules.stat_name_handler_not_supported.conf", test_conf)
        with mock_plugin_manager(plugins=plugins), mock_conf:
            msg = rule.check()

>           assert msg == expected
E           AssertionError: assert None == ('stat_name_handler field is no longer supported in AirflowTestPlugin. '\n 'stat_name_handler option in [scheduler] section in airflow.cfg should be '\n 'used to achieve the same effect.')
E             +None
E             -'stat_name_handler field is no longer supported in AirflowTestPlugin. stat_name_handler option in [scheduler] section in airflow.cfg should be used to achieve the same effect.'
```

--- a/tests/upgrade/rules/test_logging_configuration.py
+++ b/tests/upgrade/rules/test_logging_configuration.py
@@ -39,7 +39,7 @@ class TestLoggingConfigurationRule(TestCase):
             ("core", "task_log_reader"): "DUMMY",
         }
     )
-    def test_invalid_check(self):
+    def test_check(self):
         rule = LoggingConfigurationRule()
 
         assert isinstance(rule.description, str)
@@ -76,28 +76,3 @@ class TestLoggingConfigurationRule(TestCase):
         ]
         response = rule.check()
         assert response == msg
-
-    @conf_vars(
-        {
-            ("logging", "base_log_folder"): "DUMMY",
-            ("logging", "remote_logging"): "DUMMY",
-            ("logging", "remote_log_conn_id"): "DUMMY",
-            ("logging", "remote_base_log_folder"): "DUMMY",
-            ("logging", "encrypt_s3_logs"): "DUMMY",
-            ("logging", "logging_level"): "DUMMY",
-            ("logging", "fab_logging_level"): "DUMMY",
-            ("logging", "logging_config_class"): "DUMMY",
-            ("logging", "colored_console_log"): "DUMMY",
-            ("logging", "simple_log_format"): "DUMMY",
-            ("logging", "task_log_prefix_template"): "DUMMY",
-            ("logging", "log_processor_filename_template"): "DUMMY",
-            ("logging", "task_log_reader"): "DUMMY",
-        }
-    )
-    def test_valid_check(self):
-        rule = LoggingConfigurationRule()
-        assert isinstance(rule.description, str)
-        assert isinstance(rule.title, str)
-
-        response = rule.check()
-        assert response == []

